ecommerce/extensions/payment/views/hyperpay.py

```diff
@@ -42,7 +86,16 @@ class HyperPayResponseView(EdxOrderPlacementMixin, View):
     """
     Handle the response from HyperPay after processing the payment.
     """
-    SUCCESSFUL_TRANSACTION_RESULT_CODES_REGEX = re.compile(r'^(000\.000\.|000\.100\.1|000\.[36])')
+    SUCCESS_CODES_REGEX = re.compile(r'^(000\.000\.|000\.100\.1|000\.[36])')
+    SUCCESS_MANUAL_REVIEW_CODES_REGEX = re.compile(r'^(000\.400\.0[^3]|000\.400\.[0-1]{2}0)')
+    PENDING_CHANGEABLE_SOON_CODES_REGEX = re.compile(r'^(000\.200)')
+    PENDING_NOT_CHANGEABLE_SOON_CODES_REGEX = re.compile(r'^(800\.400\.5|100\.400\.500)')
```

Comment:
Can you add a comment with a link to the HyperPay documentation here?

Reply:
Resolved in ad32b8f4acbd0b4f68d93aa62ac847a82a2820c7